components/webgpu/Cargo.toml

```diff
@@ -17,5 +17,5 @@ log = "0.4"
 malloc_size_of = { path = "../malloc_size_of" }
 serde = "1.0"
 servo_config = {path = "../config"}
-smallvec = "0.6"
+smallvec = { version = "0.6", features = ["serde"] }
```

Comment:
Could smallvec be updated to 1.x in these components as well?

Reply:
I'd pretty to make that change separately.